### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by jokedurnez**
For commit [ba5229b](https://github.com/jokedurnez/HCPPipelines/commit/ba5229b9bd58c9afcaf137641c54087a44fc8f17), see [issue on hcp pipelines](https://github.com/Washington-University/Pipelines/issues/30) github .

### CODE REVIEW

**Comment by chrisfilo** [comment]
Is this ready to merge? Could you turn on CircleCI on your fork so we could see if the tests are passing?

File: run.py [hunk 1]
```diff
@@ -203,7 +201,8 @@ def run_diffusion_processsing(**args):
 
 #
 #
-run("bids-validator " + args.bids_dir)
+#run("bids-validator " + args.bids_dir)
```
Comment (chrisfilo) [inline run.py]: the validator should work after recent changes

File: run.py [hunk 2]
```diff
@@ -21,7 +21,6 @@ def run(command, env={}, cwd=None):
     while True:
         line = process.stdout.readline()
         line = str(line)[:-1]
-        print(line)
```
Comment (chrisfilo) [inline run.py]: is there a reason why you want to silence this output?

**Comment by jokedurnez** [comment]
I seem to have a problem with circleCI for the docker file...
https://circleci.com/gh/jokedurnez/HCPPipelines

**Comment by chrisfilo** [comment]
I think it's a problem with the example dataset (and poor error reporting):

```
Traceback (most recent call last):

  File "/run.py", line 253, in <module>

    t1_spacing = layout.get_metadata(t1ws[0])["EffectiveEchoSpacing"]

KeyError: 'EffectiveEchoSpacing'
```

**Comment by jokedurnez** [comment]
Well I changed some parameter in run.py from`RealDwellTime` to `EffectiveEchoSpacing`, as the current bids specification don't mention `RealDwellTime`.  This should probably then also be changed in the example data.

**Comment by chrisfilo** [comment]
I uploaded new test files: 

hcp example
https://files.osf.io/v1/resources/9q7dv/providers/osfstorage/58a1fd039ad5a101f23cd94f

lifespan example:
https://files.osf.io/v1/resources/9q7dv/providers/osfstorage/58a1fc4eb83f6901f3b47a54

Could you replace the links in the circle.yml file?

**Comment by chrisfilo** [comment]
Try rebuilding without cache on CircleCi

**Pull request merged**